posthog/auth.py

```diff
@@ -89,7 +89,7 @@ def authenticate(cls, request: Union[HttpRequest, Request]) -> Optional[Tuple[An
             key_last_used_at.day,
             key_last_used_at.hour,
         ):
-            key_last_used_at = now
+            personal_api_key_object.last_used_at = now
             personal_api_key_object.save()
```

Comment:
non-blocking

Can we make this update just last_used_at rather than the whole object? Seems like a bit of a waste to do the whole lot.

Reply:
off topic and non-blocking

It would also be nice to have some jitter in the time comparison.

Reply:
Given that this is very much in the hot path, easy to do, and we probably won't get back to it unless we have problems again - I'd probably make it blocking :D

Reply:
For the jitter, simply using a date delta being greater than an hour would probably give something of a sensible distribution.

Reply:
yes